CHANGELOG.md

```diff
@@ -933,7 +932,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Changed
 
-- Generate multiple `XXX-Project` schemes if there are multiple platforms [#2081](https://github.com/tuist/tuist/pull/2081) by [@fortmarek](https://github.com/fortmarek)
```

Comment:
Maybe let's not rename old reference in the changelog?

Reply:
oops